db/migrations/20230214142349_add_count_to_repositories.up.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
+
+alter table repositories add column failed_introspections_count int default 0 not null;
+
+COMMIT;
```

Comment:
apparently there's a migration dated after this one (2-15), and our db migrator won't actually migrate this one if that the later one is already applied.  I guess it doesn't support out-of-order migrations like rails does.  So locally this never got applied, but renaming it to 20230216...  fixed it.